Tools/scripts/install-travis-env.sh

```diff
@@ -19,8 +19,8 @@ UBUNTU64_PKGS="libc6:i386 libgcc1:i386 gcc-4.6-base:i386 libstdc++5:i386 libstdc
 
 # GNU Tools for ARM Embedded Processors
 # (see https://launchpad.net/gcc-arm-embedded/)
-ARM_ROOT="gcc-arm-none-eabi-4_7-2014q2"
-ARM_TARBALL="$ARM_ROOT-20140408-linux.tar.bz2"
+ARM_ROOT="gcc-arm-none-eabi-4_9-2015q3"
+ARM_TARBALL="$ARM_ROOT-20150921-linux.tar.bz2"
```

Comment:
You should not be building px4-v2 on this master-AVR branch.  Its only for AVR builds.  The ARM based builds (px4) have moved on significantly since this branch was created with many new features and bug fixes that this branch doesn't have.